### PR TITLE
Polishing Changes to the Config Automation

### PR DESCRIPTION
**Pull request by Abby-Wheelis**
Changes to polish the config automation including:

- [x] update commits to only include `/configs` files
- [x] adding link to reference the action I based mine on
- [x] ensuring the LISENCE is correct
- [ ] potential work on language handling
- [ ] example trip survey that's in `e-mission` (currently in @sebastianbarry's repo)

(tasks from review of #46)

### CODE REVIEW

**Comment by Abby-Wheelis** [comment]
In [d39aa9a](https://github.com/e-mission/nrel-openpath-deploy-configs/pull/49/commits/d39aa9a3a5c4e4a162dbe118ca083680e5fd1c67) and [b3e217f](https://github.com/e-mission/nrel-openpath-deploy-configs/pull/49/commits/b3e217f43ad1ab9df0259526df863957ffc1d7b7) I copied the file from @sebastianbarry's repo into this PR, and updated the link to reflect where that sample file will rest in the `e-mission` repo once this is merged. 

The thing I don't have is the `xml` or `xls` files for that survey, which might be easier to read if someone was trying to reference them. @sebastianbarry (and maybe @JGreenlee who also had commits in the repo where I found the survey), do either of you happen to still have either of those original files? Maybe they had a different name in the repo but I couldn't find them.

**Comment by JGreenlee** [comment]
There's some chance @asiripanich would have the original XML or Excel files for the trip-end-survey? It was many months ago so it's understandable if not.

**Comment by asiripanich** [comment]
@AlirezaRa94 is 

> There's some chance @asiripanich would have the original XML or Excel files for the trip-end-survey? It was many months ago so it's understandable if not.

@AlirezaRa94, can you please help @JGreenlee with this? I think it can be easily exported from our Enketo account.

**Comment by AlirezaRa94** [comment]
@asiripanich 
@JGreenlee 
Is this the file that you are looking for?
[trip-confirm-survey-form.zip](https://github.com/e-mission/nrel-openpath-deploy-configs/files/13520233/trip-confirm-survey-form.zip)

**Comment by Abby-Wheelis** [comment]
@AlirezaRa94 yes, that matches the JSON file that we have, thank you! If you happen to also have the xlsx file that would be great, but if not no worries!

**Comment by Abby-Wheelis** [comment]
Screenshot 2023-12-06 at 9 14 51 AM
Screenshot 2023-12-06 at 9 15 33 AM
 

Some examples of what the new field looks like once the issue has been created, and in the issue form.

File: .github/ISSUE_TEMPLATE/add-new-config.yml [hunk 1]
```diff
@@ -506,6 +506,13 @@ body:
       - "false"
   validations:
     required: true  
+- type: textarea
+  id: admin_access
+  attributes:
+    label: Admin Access Emails
+    description: Please enter a comman separated list of emails to which we will grant access to the admin dashboard. This will usually be a relatively short list.
```
Comment (shankari) [inline .github/ISSUE_TEMPLATE/add-new-config.yml:513]: We used to have a limit of 3 users in the appendix. Maybe we can add a check for this number in the validation and limit it to less than 5?

File: .github/actions/convertIssue/action.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # this action is based on the action by zachleat on github (github-issue-to-json-file)
+# https://github.com/zachleat/github-issue-to-json-file/blob/main/action.yml
```
Comment (shankari) [inline .github/actions/convertIssue/action.yml:2]: @Abby-Wheelis in general, we cannot use copy that does not explicity have a LICENSE. We can "adapt" based on it if the changes are sufficiently large. You might want to check with legal on whether this meets that threshold.

**Review by shankari — APPROVED** [review]

**Comment by shankari** [comment]
Better than before, needs some check/follow-up on the LICENSE

**Pull request merged**